tests/baselines/reference/autoAccessorExperimentalDecorators(target=esnext).errors.txt

```diff
@@ -0,0 +1,27 @@
+tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessorExperimentalDecorators.ts(12,5): error TS1206: Decorators are not valid here.
+tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessorExperimentalDecorators.ts(15,5): error TS1206: Decorators are not valid here.
+
+
+==== tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessorExperimentalDecorators.ts (2 errors) ====
+    declare var dec: (target: any, key: PropertyKey, desc: PropertyDescriptor) => void;
+    
+    class C1 {
+        @dec
+        accessor a: any;
+    
+        @dec
+        static accessor b: any;
+    }
+    
+    class C2 {
+        @dec
+        ~
+!!! error TS1206: Decorators are not valid here.

```

Comment:
will ES decorators be valid here

Reply:
Yes. Legacy decorators aren't supported on members with private identifiers, but will be supported for ES decorators.